src/types/rendering.ts

```diff
@@ -15,6 +8,13 @@ export interface DOMCoordinates {
   y: Pixel;
 }
 
+export interface PixelDimension {
```

Comment:
Just noticed we already have `DOMDimension`, guess we can stick with it

Reply:
Indeed, it is done :)